Curtains/Curtains.Application/CurtainsService/NotifyService.cs

```diff
@@ -0,0 +1,105 @@
+using AutoMapper;
+using Curtains.Application.ConstructorObjects;
+using Curtains.Application.ConstructorObjects.Interfaces;
+using Curtains.Application.CurtainsService.Interfaces;
+using Curtains.Application.DTO;
+using Curtains.Infrastructure.Shared.Exceptions;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using System.Text.Json;
+
+namespace Curtains.Application.CurtainsService
+{
+    /// <summary>
+    /// Notifies the customer of an order
+    /// </summary>
+    class NotifyService : INotifyService
+    {
+        private const string _subject = "Вам поступил новый заказ!";
+        private readonly IMessageSender _messageSender;
+        private readonly IMessageConstructor _messageBuilder;
```

Comment:
Why are you resolve comments but not answer or make changes?

Reply:
my bad

Reply:
done :)